frontend/src/hooks/api/actions/usePersonalAPITokensApi/usePersonalAPITokensApi.ts

```diff
@@ -37,9 +37,26 @@ export const usePersonalAPITokensApi = () => {
         }
     };
 
+    const createUserPersonalAPIToken = async (
+        userId: number,
+        payload: ICreatePersonalApiTokenPayload
+    ): Promise<INewPersonalAPIToken> => {
+        const req = createRequest(`api/admin/user-admin/${userId}/pat`, {
+            method: 'POST',
+            body: JSON.stringify(payload),
+        });
+        try {
+            const response = await makeRequest(req.caller, req.id);
+            return await response.json();
+        } catch (e) {
+            throw e;
+        }
```

Comment:
Out of curiosity, wouldn't this be the same?
```suggestion
        const response = await makeRequest(req.caller, req.id);
        return await response.json();
```

Reply:
You're right that it should be the same, but this is consistent with the other hooks we have. No idea what the original reasoning was behind it.